### PR TITLE
Translate performance-and-scalability/scenarios into Chinese

### PR DESCRIPTION
**Pull request by xiaosuiba**
[original code](https://github.com/istio/istio.github.io/tree/master/content/docs/performance-and-scalability/scenarios/index.md)
[original page](https://preliminary.istio.io/docs/performance-and-scalability/scenarios)

### CODE REVIEW

File: content_zh/docs/performance-and-scalability/scenarios/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: 测试场景
+description: 我们正在跟踪的性能和可伸缩性的不同场景。
```
Comment (gyliu513) [inline content_zh/docs/performance-and-scalability/scenarios/index.md]: 我建议把”可伸缩性“改为”规模性测试”

Comment (gyliu513) [reply content_zh/docs/performance-and-scalability/scenarios/index.md]: after some discussion in wechat group, we agree to use "可伸缩性"

Comment (xiaosuiba) [reply content_zh/docs/performance-and-scalability/scenarios/index.md]: ok, I see it.

**Pull request merged**